### PR TITLE
Better find_events message if no stim channel is found

### LINKED ISSUE #6345: No stim channels found. Consider specifying them manually using the 'stim_channel' parameter.
Hi,
I got an error when I call 
_mne.find_events(raw=raw, shortest_event=1, verbose=False)_ that I didn't get previously:

_No stim channels found. Consider specifying them manually using the 'stim_channel' parameter._

Why is that? I went back to version 0.17.2

### PR DESCRIPTION
**Pull request by cbrnr**
Fixes #6345. OK @agramfort?

### CODE REVIEW

File: mne/utils/config.py [hunk 1]
```diff
@@ -406,7 +406,11 @@ def _get_stim_channel(stim_channel, info, raise_error=True):
         stim_channel = [info['ch_names'][ch_] for ch_ in stim_channel]
     elif raise_error:
         raise ValueError("No stim channels found. Consider specifying them "
-                         "manually using the 'stim_channel' parameter.")
+                         "manually using the 'stim_channel' parameter. In "
+                         "many file formats, discrete markers are now "
+                         "available in raw.annotations. Consider using "
+                         "mne.events_from_annotations to convert these to "
+                         "events.")
```
Comment (agramfort) [inline mne/utils/config.py]: I would check that some annotations are present before tell people to use events_from_annotations

Comment (cbrnr) [reply mne/utils/config.py]: Alright. I even added a test 😄.

**Comment by larsoner** [comment]
Thanks @cbrnr

**Comment by agramfort** [comment]
thx @cbrnr

File: mne/event.py [hunk 2]
```diff
@@ -678,7 +678,16 @@ def find_events(raw, stim_channel=None, output='onset',
     min_samples = min_duration * raw.info['sfreq']
 
     # pull stim channel from config if necessary
-    stim_channel = _get_stim_channel(stim_channel, raw.info)
+    try:
+        stim_channel = _get_stim_channel(stim_channel, raw.info)
+    except ValueError:
+        if len(raw.annotations) > 0:
```
Comment (massich) [inline mne/event.py]: `if raw.annotations:` should work.

**Pull request merged**